tools/tinytest-codegen.py

```diff
@@ -48,6 +48,7 @@ def script_to_map(t):
 # currently these tests are selected because they pass on qemu-arm
 test_dirs = ('basics', 'micropython', 'float', 'extmod', 'inlineasm') # 'import', 'io', 'misc')
 exclude_tests = (
+    'basics/class_reverse_op.py',
```

Comment:
qemu-arm is used for testing on ARM Thumb2, so makes sense to enable this feature and the test, rather than exclude it.

Reply:
Done